src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx

```diff
@@ -57,7 +57,9 @@ Here's a screenshot of the **User management** UI. We'll explain the column head
 />
 
 * **Name**: The user's name. 
-* **Email**: The user's email address. `Pending` indicates a user has been added to New Relic but has not yet logged in. `Pending` users are still considered provisioned: if they have a billable user type they are billable regardless of their New Relic activity. 
+* **Email**: The user's email address. Some notes on messages you may see present: 
```

Comment:
probably swap the "if they have" and "they are billable" parts of the last sentence for Pending